packages/web/src/screens/Signin.tsx

```diff
@@ -1,98 +1,112 @@
-import React, { Component } from 'react';
-import styled from 'styled-components';
-import Input from '../components/Input';
-import Button from '../components/Button';
-import { graphql } from 'react-apollo';
-import gql from 'graphql-tag'
+import React from "react";
+import styled from "styled-components";
+import Input from "../components/Input";
+import Button from "../components/Button";
+import { graphql } from "react-apollo";
+import gql from "graphql-tag";
 
 const loginUser = gql`
-  mutation loginUser($input: LoginUserInput!) {
-    loginUser(input: $input) {
-      token
-      error
+    mutation loginUser($input: LoginUserInput!) {
+        loginUser(input: $input) {
+            token
+            error
+        }
     }
-  }
-`
+`;
```

Comment:
sometimes we're using a mutation inside a file and smetimes not, can we create a pattern?